ember-flight-icons/addon/components/flight-icon.hbs

```diff
@@ -1,7 +1,7 @@
 <svg 
```

Comment:
Sorry I missed this earlier. You also need to set `role={{if @title "img" "presentation"}}` for the title attribute to work correclty.

Reply:
This last one will be a blocker; I'm going to put this back into draft mode until the DS team has time to discuss further. 

Thank you for the great feedback @junyper!